nautobot/core/graphql/schema.py

```diff
@@ -159,6 +167,34 @@ def extend_schema_type_null_field_choice(schema_type, model):
     return schema_type
 
 
+def extend_schema_type_filter(schema_type, model):
+    """Extend schema_type object to be able to filter on multiple levels of a query
+
+    Args:
+        schema_type (DjangoObjectType): GraphQL Object type for a given model
+        model (Model): Django model
+
+    Returns:
+        schema_type (DjangoObjectType)
+    """
+    for field_name in dir(model):
+        attr = getattr(model, field_name)
```

Comment:
Question of consistency here - in other `extend_*` methods we're either doing `for field in model._meta.fields` or `for field in model._meta.get_fields()` - would either of those be more consistent/correct here?